docs/manager.md

```diff
@@ -0,0 +1,254 @@
+# Deploying Scylla Manager on a Kubernetes Cluster
+
+Scylla Manager is a product for database operations automation, 
+it can schedule tasks such as repairs and backups. 
+Scylla Manager can manage multiple Scylla clusters and run cluster-wide tasks 
+in a controlled and predictable way.
+
+Scylla Manager is available for Scylla Enterprise customers and Scylla Open Source users. 
+With Scylla Open Source, Scylla Manager is limited to 5 nodes. 
+See the Scylla Manager [Proprietary Software License Agreement](https://www.scylladb.com/scylla-manager-software-license-agreement/) for details.
+
+## Prerequisites
+
+* Kubernetes cluster
+* Scylla Operator - see [generic guide](generic.md)
+
+## Architecture
+
+Scylla Manager in K8s consist of:
+- Dedicated Scylla Cluster
+
+  Scylla Manager persists its state to a Scylla cluster. 
```

Comment:
Somewhat unrelated, but how big is the state?
Do we plan for a closer integration with Kubernetes in the future? If so, could it store its state in etcd?

Reply:
It depends on how much clusters you manage and how much tasks there are. We do have TTLs on task progress table which should be biggest. I don't have a numbers though.

We thought about adding different state layer, but we don't have usecase which can't be covered with current architecture. So nothing in the pipeline.